.travis.yml

```
@@ -8,14 +8,11 @@ env:
     - AWS_DEFAULT_REGION=$AWS_REGION
 install:
   - pip3 install --user pre-commit cloudformation-cli-java-plugin
-script:
+before_script:
   - pre-commit run --all-files --verbose
-  - cd "$TRAVIS_BUILD_DIR/aws-logs-loggroup"
-  # from Maven 3.6.1+, should use `--no-transfer-progress` instead of Slf4jMavenTransferListener
-  - >
-      mvn
-      -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-      -B
-      clean verify
-after_failure:
-  - cat "$TRAVIS_BUILD_DIR/aws-logs-loggroup/rpdk.log"
+jobs:
+  include:
+    - script:
+        - for directory in $TRAVIS_BUILD_DIR/aws-*; do (cd "$directory" && mvn clean verify); done
```

[left all the arguments out of my comment showing the general structure:](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/pull/42#issuecomment-651423690)
```suggestion
        - for directory in $TRAVIS_BUILD_DIR/aws-*; do (cd "$directory" && mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean verify); done
```